tests/subworkflows/ebi-metagenomics/fasta_domainannotation/test.yml

```diff
@@ -0,0 +1,37 @@
+- name: fasta_domainannotation_diamond
+  command: nextflow run ./tests/subworkflows/ebi-metagenomics/fasta_domainannotation/main.nf -entry test_fasta_domainannotation_diamond -c ./tests/config/nextflow.config -c ./tests/subworkflows/ebi-metagenomics/fasta_domainannotation/nextflow.config
+  tags:
+    # - blast
```

Comment:
Please remove the unused tags.

Reply:
They shall be uncommented (need to be there) before being uploaded in nf-core. For now they introduced a bug in the tests which I need to recheck. Please ignore

Reply:
Ok they finally all pased with nf-tests. Should be good.